pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1434,6 +1516,57 @@
     ^+  event-core
     %-  %^  trace  odd.veb  sndr.packet
         |.("received packet")
+    :: add the ship to our sponsees map if we are the sponsor
+    ::
+    =*  relay-state  relay-state.ames-state
+    =?  sponsees.relay-state
+      ?&
+        =(~ next.packet)
+        =/  val  (~(get by sponsees.relay-state) sndr.packet)
+        =(~ val)
+        =(our (sein:title our now sndr.packet))
+      ==
+      ::  send 30s expiration timer to behn
+      ::
+      =/  duct  ~[/ames]
+      =/  wire  (make-sponsee-timer-wire sndr.packet)
+      =.  event-core
+        (emit duct %pass wire %b %wait (add ~s30 now))
+      ::
+      ^+  sponsees.relay-state
+      (~(put by sponsees.relay-state) [sndr.packet [lane (add ~s30 now)]])
+    :: add the lane to our pending map
+    ::
+    =*  inv-req  [%ames `dyad`[rcvr.packet sndr.packet]]
+    =?  pending.relay-state
+      =/  val  (~(get by pending.relay-state) inv-req)
+      ?~  val  %.y
+        =/  next  q.-.u.val
```

Comment:
style: dedent these four lines, since it's the second option of the `?~  val` and should therefore be at the same indentation level as the `?~` to maintain backstep